src/OrbitGl/App.cpp

```diff
@@ -1515,6 +1528,14 @@ void OrbitApp::StartCapture() {
   Future<ErrorMessageOr<CaptureOutcome>> capture_result = capture_client_->Capture(
       thread_pool_.get(), std::move(capture_event_processor), *module_manager_, options);
 
+#ifdef _WIN32
```

Comment:
But this is conceptually wrong. This is about where the service is. When the service is on Linux and the client is on Windows, which is the regular case, this code still gets executed. If we are fine with it because it has no effect, then there should be no ifdefs at all.